src/mixins/wordWrap.js

```diff
@@ -0,0 +1,33 @@
+/** @module Mixins */
+
+/**
+ * Provides an easy way to change the `word-wrap` property
+ *
+ * @static
+ * @param {string} [wrap = 'break-word'] - Value for the word-break property
+ * @return {Object} CSS as JS.
+ * @example
+ * div {
+ *   wordWrap('break-all')
+ * }
+ *
+ * // CSS as JS Output
+ *
+ * div {
+ *   overflow-wrap: 'break-all',
+ *   word-wrap: 'break-all',
+ *   word-break: 'break-all',
+ * }
+ */
+
+function wordWrap(wrap = 'break-word') {
+  const wordBreak = wrap === 'break-word' ? 'break-all' : wrap
+  return {
+    'overflow-wrap': wrap,
+    'word-wrap': wrap,
+    'word-break': wordBreak,
+  }
+}
+
```

Comment:
Something I've been thinking about based on how Bourbon, Compass, etc.. are often used is how do we account for this traditional usage:

```scss
div {
  background: red;
  @include word-wrap;
}
```

Obviously this syntax doesn't work 1 to 1 in JS, but we should decide on how we want to tell users to handle this scenario. With this current implementation, we would have no way to in handle this.